### PR TITLE
goatcorp.XIVLauncher version 5.5.8

### PR DESCRIPTION
**Pull request by avafloww**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19001)

### CODE REVIEW

File: manifests/g/goatcorp/XIVLauncher/5.5.8/goatcorp.XIVLauncher.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: goatcorp.XIVLauncher
+PackageVersion: 5.5.8
+PackageLocale: en-US
+Publisher: goatcorp
```
Comment (ImJoakim) [inline manifests/g/goatcorp/XIVLauncher/5.5.8/goatcorp.XIVLauncher.locale.en-US.yaml:7]: image


```suggestion
Publisher: goaaats
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see https://github.com/microsoft/winget-pkgs/pull/19001/files#r659266405.

Windows Package Manager requires `Publisher` to match whatever's shown in Control Panel to properly map the application in `winget list` and allow other people to upgrade the application via the `winget upgrade` command.

**Comment by avafloww** [comment]
> 
> 
> Please see https://github.com/microsoft/winget-pkgs/pull/19001/files#r659266405.
> 
> Windows Package Manager requires `Publisher` to match whatever's shown in Control Panel to properly map the application in `winget list` and allow other people to upgrade the application via the `winget upgrade` command.

I see, TIL. Updated!

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by TWinn22** [comment]
Where is the source for the Goatcorp.FFXIVlauncher? What enhancements does this add to the game, etc.?

**Comment by ItzLevvie** [comment]
You can read that here: https://github.com/goatcorp/FFXIVQuickLauncher#why.

**Pull request merged**